README.md

```diff
@@ -58,33 +58,36 @@ If you clone this repository, use the checkout commands to access the sources fo
 | Name                                                                                                   | Container          |
 | -------------------------------------------------------------------------------------------------------|--------------------|
 | [Simple Process Applications](https://docs.camunda.org/get-started/bpmn20/)                            | All                |
-| [Camunda Platform and the Spring Framework](https://docs.camunda.org/get-started/spring/)              | Tomcat             |
-| [Process Applications with JavaEE 6](https://docs.camunda.org/get-started/javaee6/)                    | JavaEE Containers  |
+| [Camunda Platform and the Spring Framework](https://docs.camunda.org/get-started/spring/) (1)          | Tomcat             |
+| [Process Applications with JavaEE 6](https://docs.camunda.org/get-started/javaee6/) (1)                | JavaEE Containers  |
+
+(1) _This example is not actively maintained anymore._
```

Comment:
🔧 Alternatively, we could use real clickable footnotes: [^1]

[^1]: [github blog post](https://github.blog/changelog/2021-09-30-footnotes-now-supported-in-markdown-fields/)